modules/examiner/php/editexaminer.class.inc

```diff
@@ -189,7 +189,7 @@ class EditExaminer extends \NDB_Form
             } else { // update to a test certification for the examiner
 
                 //select history events
-                $oldVals = $DB->pselectRow(
+                $oldVals = $DB->pselect(
```

Comment:
Since this is explicitly using an `ORDER BY` clause and only getting the first row from the result, it should also add a `LIMIT 1` (which pselectRow does implicitly).

Can you also add a comment above the line where you access `$oldVals[0]` explaining that, since there's an `ORDER BY`, the value will always be consistent?